test/tasks/ods-build-java_test.go

```diff
@@ -0,0 +1,89 @@
+package tasks
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/opendevstack/pipeline/internal/command"
+	"github.com/opendevstack/pipeline/pkg/tasktesting"
+)
+
+func TestTaskODSBuildJava(t *testing.T) {
+	runTaskTestCases(t,
+		"ods-build-java-v0-1-0",
+		map[string]tasktesting.TestCase{
+			"task should build java/maven app": {
+				WorkspaceDirMapping: map[string]string{"source": "java-sample-app"},
+				PreRunFunc: func(t *testing.T, ctxt *tasktesting.TaskRunContext) {
+					wsDir := ctxt.Workspaces["source"]
+					ctxt.ODS = tasktesting.SetupGitRepo(t, ctxt.Namespace, wsDir)
+					ctxt.Params = map[string]string{
+						"java-image":    "localhost:5000/ods/ods-java-toolset:latest",
+						"sonar-image": "localhost:5000/ods/ods-sonar:latest",
```

Comment:
With the latest changes in `master`, it is not necessary anymore to specify the images as parameters :) Please remove these 2 lines.

Reply:
ok